Sources/StreamChatUI/ChatMessageList/ChatMessageListCollectionViewLayout.swift

```diff
@@ -277,11 +277,13 @@ open class ChatMessageListCollectionViewLayout: UICollectionViewLayout {
         super.prepare()
 
         guard !didPerformInitialLayout else { return }
-        didPerformInitialLayout = true
 
         guard currentItems.isEmpty else { return }
         guard let cv = collectionView else { return }
         currentCollectionViewWidth = cv.bounds.width
+        
+        guard currentCollectionViewWidth != 0 else { return }
```

Comment:
For `ChatThreadVC` after the initial preparation the width is zero which lead to this:
![image](https://user-images.githubusercontent.com/7489425/121627178-8d95e380-ca7f-11eb-8750-43c5a83f383e.png)
I wasn't able to figure out why this happens for `ChatThreadVC` but not for `ChatMessageListVC`, but we agreed with @VojtaStavik that this fix shouldn't hurt